pyproject.toml

```diff
@@ -18,12 +18,11 @@ numpyro = "^0.13.2"
 blackjax = "^0.9.6"
 matplotlib = "^3.9"
 seaborn = "^0.13.2"
+pyyaml = ">=5.3"
```

Comment:
I don't think version pins should be based on transitive dependencies – it should be on direct usage by the project. evofr is just calling `yaml.safe_load`, so `"^5.0` or even `"^5.0 || ^6.0"` should be fine.